p2p/validation/msg_validation.go

```diff
@@ -14,28 +14,13 @@ import (
 // MsgValidatorFunc represents a message validator
 type MsgValidatorFunc = func(ctx context.Context, p peer.ID, msg *pubsub.Message) pubsub.ValidationResult
 
-// To-Do: better spec the intended Message Validation behavior. Issue #375: https://github.com/bloxapp/ssv-spec/issues/375
-type MessageValidator interface {
-	ValidateSignedSSVMessage(runner ssv.Runner, signedSSVMessage *types.SignedSSVMessage) error
-	// Verifies the message's RSA signature with PKCSv1.5 encoding. Ref: https://datatracker.ietf.org/doc/html/rfc8017#section-8.2.2
-	VerifySignature(runner ssv.Runner, signedSSVMessage *types.SignedSSVMessage) error
-}
-
-func MsgValidation(runner ssv.Runner, msgValidator MessageValidator) MsgValidatorFunc {
+func MsgValidation(runner ssv.Runner) MsgValidatorFunc {
```

Comment:
do you see value to keep this file here for now?
if no, you may delete it

Reply:
doesn't have to be in this PR and not a blocker

Reply:
Future PR